docs/tooling.md

```diff
@@ -26,15 +28,25 @@ The original file is preserved and a new file is created with `-reduced` appende
 
 ---
 
-## Analyse XML command
+## Analyse XML commands
 
-This is a command to help you analyse the data you will be importing.
+These commands help you analyse the XML data source.
+
+### analyze_html_content
+
+Will generate a table output in the console to show inline styles, HTML tags and shortcodes used in the `<content:encoded>` XML tag.
 
 ```bash
 python manage.py analyze_html_content path/to/your/xmlfile.xml
 ```
 
-It will generate a table output in the console to show inline styles, HTML tags and shortcodes used to format the content.
+### analyze_xml_content
+
+Will generate a JSON file your project which shows the structure of all unique XML tags and attributes.
```

Comment:
**Suggestion**: Mention the fact that the input file path determines the path of the JSON file.